pkg/user/utils/txclient_utils.go

```diff
@@ -0,0 +1,93 @@
+package utils
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v6/app"
+	"github.com/celestiaorg/celestia-app/v6/app/encoding"
+	"github.com/celestiaorg/celestia-app/v6/pkg/user"
+	"github.com/celestiaorg/celestia-app/v6/test/util/genesis"
+	"github.com/celestiaorg/celestia-app/v6/test/util/testfactory"
+	"github.com/celestiaorg/celestia-app/v6/test/util/testnode"
+	blobtypes "github.com/celestiaorg/celestia-app/v6/x/blob/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	sdktx "github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/stretchr/testify/require"
+)
+
+func SetupTxClient(
+	t *testing.T,
+	ttlNumBlocks int64,
+	squareSize uint64,
+	blocksize int64,
+	opts ...user.Option,
+) (encoding.Config, *user.TxClient, testnode.Context) {
+	enc := encoding.MakeConfig(app.ModuleEncodingRegisters...)
+	defaultTmConfig := testnode.DefaultTendermintConfig()
+	defaultTmConfig.Mempool.TTLNumBlocks = ttlNumBlocks
```

Comment:
[nit] this isn't the defaultTmConfig any more if the function overrides a variable


```suggestion
	tmConfig := testnode.DefaultTendermintConfig()
	tmConfig.Mempool.TTLNumBlocks = ttlNumBlocks
```